app/src/main/java/com/group2/catan_android/fragments/TradingResourceSelectionFragment.java

```diff
@@ -64,51 +70,19 @@ public void setPlusAndMinusListener(View view){
         for(int i=0;i<plus.length;i++){
             final int j = i;
             plus[j].setOnClickListener(v -> {
-                int num = getNumberofTextView(count[j]);
-                if (num == -1) {
-                    return;
-                }//Error
-                count[j].setText(Integer.toString(++num));
+                count[j].setText(Integer.toString(tradeResourcesViewModel.togglePlus(j)));
```

Comment:
Do what the compiler says. Also in general, if a warning is shown, make it go away

Reply:
changed to String.format()